plugins/modules/cloud/misc/terraform.py

```diff
@@ -27,6 +27,15 @@
       - The path of a terraform binary to use, relative to the 'service_path'
         unless you supply an absolute path.
     type: path
+  declare_variables:
+    description:
+      - If true, generate a Terraform file that declares all variables supplied via this module's
+        'variables' parameter. Type, description, and default value are not specified.
```

Comment:
```suggestion
        I(variables) parameter. Type, description, and default value are not specified.
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation for details on formatting.